Cmdr/BuiltInCommands/help.lua

```diff
@@ -1,3 +1,13 @@
+local ARGUMENT_SHORTHANDS = ([[
+Argument Shorthands
+-------------------			
+.   Me/Self
+*   All/Everyone 
+**  Others 
+?   Random
+?N  List of N random values			
+]])
```

Comment:
Excess parenthesis surrounding multiline string literal

```suggestion
local ARGUMENT_SHORTHANDS = [[
Argument Shorthands
-------------------			
.   Me/Self
*   All/Everyone 
**  Others 
?   Random
?N  List of N random values			
]]
```